docs/index.md

```diff
@@ -96,6 +97,23 @@ $ export HW_REGION_NAME="cn-north-4"
 $ terraform plan
 ```
 
+### Shared Configuration File
+
+You can use a
+[HuaweiCloud CLI configuration file](https://support.huaweicloud.com/intl/en-us/usermanual-hcli/hcli_03_002.html)
+to specify your credentials. You need to specify a location in the Terraform configuration by providing the
+`shared_configuration_file` argument or using the `HW_SHARED_CONFIGURATION_FILE` environment variable.
+This method also supports a `profile` configuration and matching `HW_PROFILE` environment variable:
+
+Usage:
+
+```terraform
+provider "huaweicloud" {
+  shared_configuration_file = "/Users/tf_user/.hcloud/config.json"
```

Comment:
Generally, the home-dir of a user is **/home/{{user_name}}**

Reply:
done